easybuild/easyconfigs/c/CMake/CMake-3.6.1.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.6.1'
+
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = '-- -DCMAKE_USE_OPENSSL=1'
+
+# Don't depend on anything, since we will use this to build other software
+# which might want different versions of the same dependency, eg: ncurses
+dependencies = []
```

Comment:
@valtandor I guess specifying `dependencies = []` doesn't make much sense

Maybe you should replace this block with a comment above `osdependencies` like this:

```python
# use OS dependencies to ensure that CMake can be used to build software that depends on them
```